src/elements/profile/user-banner.ts

```diff
@@ -1,95 +1,37 @@
-import { html, LitElement, TemplateResult } from 'lit';
-import { customElement, property, query } from 'lit/decorators.js';
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
 import { cssify } from '../../utils/css';
-import styles from './user-banner.scss';
 import global from '../../utils/global';
-import assets from '../../data/assets';
 import { responses } from '../../routes';
 import { when } from 'lit/directives/when.js';
 import RvtRoot from '../root/rvt-root';
 
-interface Splash {
-	render: (name: string) => TemplateResult;
-	weight: number;
-}
-
-const SPLASHES: Splash[] = [
-	{
-		render: name =>
-			html`<span class="text-2xl font-semibold">Welcome to Rivet,</span><br /><span class="text-6xl font-display leading-12">${name}</span></span></span>`,
-		weight: 1
-	}
-];
-
 @customElement('user-banner')
 export default class UserBanner extends LitElement {
-	static styles = cssify(styles);
-
-	@query('video#bg')
-	videoBg: HTMLVideoElement;
-
-	@property({ type: Number })
-	windowWidth: number = window.innerWidth;
+	static styles = cssify();
 
 	@property({ type: Object })
 	loadError?: any;
 
-	splashSeed: number = Math.random();
-
-	onScroll(e: Event) {
-		let target = (e.currentTarget || e.target) as HTMLElement;
-
-		this.updateVideoPosition(target);
-	}
-
-	updateVideoPosition(target: HTMLElement) {
-		if (this.videoBg && !(target instanceof Window)) {
-			this.videoBg.style.transform = `translateY(${target.scrollTop}px)`;
-		}
-	}
-
-	onResize() {
-		this.windowWidth = window.innerWidth;
-		this.updateVideoPosition(document.body);
-	}
-
 	constructor() {
 		super();
 	}
 
 	render() {
 		if (this.loadError) return responses.renderError(this.loadError);
 
-		// Don't ever need to deliver anything above 960x540 since it's a
-		// blurred video and the content has a max width
-		let bannerVideoUrl =
-			this.windowWidth > 1200
-				? assets.asset('graphics/home-banner/banner-960x540.mp4')
-				: assets.asset('graphics/home-banner/banner-640x360.mp4');
-		let bannerVideoSpeed = 0.7;
-
-		return html`<div id="base">
-			<div id="centered-body">
-				<div id="banner" class="block">
-					<div id="banner-bg-crop">
-						<video
-							id="banner-bg"
-							src=${bannerVideoUrl}
-							autoplay
-							muted
-							disablePictureInPicture
-							disableRemotePlayback
-							loop
-							playsinline
-							.playbackRate=${bannerVideoSpeed}
-						></video>
-					</div>
-
-					<!-- Header -->
-					<div id="banner-content">
-						<identity-avatar hide-status .identity=${global.currentIdentity}></identity-avatar>
-						<div id="banner-text">
-							<div>${this.renderSplashText()}</div>
+		return html`
+			<div class="mx-auto max-w-contentwidth px-[10px] md:px-5 lg:px-0 pb-8">
+				<div class="relative my-[20px] md:my-[40px] bg-gradient-to-r from-pink-800 to-orange-600 h-[18.2rem] rounded-[20px]">
```

Comment:
Same here, try to find the closest values from tailwind theme. Doing things that way, we minimizing the chances to end up up with things that are not aligned properly.